python/paddle/fluid/learning_rate_decay.py

```diff
@@ -30,11 +30,14 @@
 """
 
 
-def exponential_decay(learning_rate,
-                      global_step,
-                      decay_steps,
-                      decay_rate,
-                      staircase=False):
+def float_global_step():
+    # the first global step is zero in learning rate decay
+    global_step = layers.global_step_counter() - 1
```

Comment:
this line is confusing.

Reply:
Done. Changed by layers.autoincrement_counter(begin=0)